src/main/java/uk/gov/hmcts/reform/civil/utils/CaseQueriesUtil.java

```diff
@@ -204,6 +205,16 @@ public static CaseMessage getQueryById(CaseData caseData, String queryId) {
             .orElseThrow(() -> new IllegalArgumentException("No query found for queryId " + queryId));
     }
 
+    private static boolean isWelshQuery(CaseData caseData, List<String> roles) {
+        String languagePreference = null;
+        if (isLIPClaimant(roles)) {
+            languagePreference = caseData.getClaimantBilingualLanguagePreference();
+        } else if (isLIPDefendant(roles)) {
+            languagePreference = caseData.getDefendantBilingualLanguagePreference();
```

Comment:
I think we can use isClaimantBilingual() and isRespondentResponseBilingual(), they should already handle nulls and WELSH or BOTH